patches/chrome-browser-resources-settings-privacy_page-privacy_page.html.patch

```diff
@@ -1,40 +1,56 @@
 diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.html b/chrome/browser/resources/settings/privacy_page/privacy_page.html
-index 750900dfc674d286bef0b3d2c5a764c6035a232c..9e0a76ef24509fcee33343b7761eadb3e2f3af17 100644
+index f58c307b46222efb8acbf2f9bf11af5cf14fd211..4c22140cbe146e5fcfea2566d84a85e2b502491e 100644
 --- a/chrome/browser/resources/settings/privacy_page/privacy_page.html
 +++ b/chrome/browser/resources/settings/privacy_page/privacy_page.html
-@@ -94,6 +94,7 @@
-             on-click="onSyncAndGoogleServicesClick_"
-             hidden="[[!unifiedConsentEnabled_]]"></cr-link-row>
+@@ -38,6 +38,7 @@
+ 
+ <if expr="not chromeos">
+ <link rel="import" href="chrome://resources/cr_elements/cr_toast/cr_toast.html">
++<link rel="import" href="../brave_rewards_privacy/brave_rewards_privacy.html">
```

Comment:
can we make this file more generic? `../brave_privacy.html` or something which will then in turn have a link to `../brave_rewards_privacy/brave_rewards_privacy.html`
The reason it that future changes to privacy_page.html may not be related to rewards, but we can add those additional links in `../brave_privacy.html` to avoid patching this file again

Reply:
Done.